### PR TITLE
upgrades: removal of legacy recipes

### LINKED ISSUE #14: upgrader: remove old upgrades
- [x] remove old upgrades
- [x] remove dependency on `invenio.legacy.dbquery:run_sql`  (cc @egabancho)

### PR DESCRIPTION
**Pull request by jirikuncar**
- INCOMPATIBLE Removes legacy upgrade recipes. You **MUST** upgrade to
  the latest Invenio 2.1 before upgrading Invenio-Upgrader.
  (closes #14)

Signed-off-by: Jiri Kuncar jiri.kuncar@cern.ch
## 

cc @egabancho

### CODE REVIEW

File: invenio_upgrader/upgrades/invenio_upgrader_2015_09_23_legacy_removal.py [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+"""Remove legacy upgrade recipes."""
+
+import warnings
+
+from invenio_ext.sqlalchemy import db
+from invenio_upgrader.api import op
+from invenio_utils.text import wait_for_user
+
+from sqlalchemy import *
```
Comment (egabancho) [inline invenio_upgrader/upgrades/invenio_upgrader_2015_09_23_legacy_removal.py]: Is this needed for some reason that I'm not aware of? (Probably yes)

**Comment by egabancho** [comment]
I think it would be nice if we could provide some kind of recipe/howto for those of us coming from Invenio 1.2, something like "install Invenio 2.1 upgrade and then install Invenio _2.9_ and uprade again"

File: invenio_upgrader/upgrades/invenio_upgrader_2015_09_23_legacy_removal.py (hunk 1 above)
Comment (jirikuncar) [inline invenio_upgrader/upgrades/invenio_upgrader_2015_09_23_legacy_removal.py]: It's not needed. It was auto-generated.

**Comment by tiborsimko** [comment]
I'd say that even v2.1 to v3.0 upgrade may be considered only "nice-to-have" WRT automatic upgrade recipes, especially because there are not many installations using v2.1 live, and because people may profit from JSON Schema based developments in order to amend their data model at the same time. Kind of taking this opportunity to start with a new clean base line...

For Invenio v1.x users, with modules like BibDocFile gone, WebSubmit gone, the upgrade to v3.0 will be largely non-automatic anyway.  People will have to adapt their ingestion workflows. It should be sufficient if we provide detailed guides as to how one would upgrade, but this can remain at the level of documentation recipes only (e.g. define your MARC schema like this, port your records like this, port your documents like this, port your collections and access rights like this, etc). This will enable everyone to have a basic v3.0 development site up and running without having to go progressively through hundreds of upgrade recipes from v1.0.x to v3.0.x that would be counter-productive in my eyes.

**Pull request merged**